code/ide.py

```diff
@@ -499,3 +511,6 @@ def ide_show_debug():
 
     def ide_show_extensions():
         """Show Extensions"""
+
+    def ide_window_new():
```

Comment:
I don't think this new action is necessary, should be able to override the talon-declared one

https://github.com/knausj85/knausj_talon/blob/master/apps/win/app.talon#L37

and we could stick with the global 'new window' command with VSCode
https://github.com/knausj85/knausj_talon/blob/606421ea22a321a22c44065de6133f396b81c1c4/misc/window_management.talon#L1

Reply:
Thanks that works